filters/cloudychat/cloudychat.go

```diff
@@ -13,17 +14,15 @@ import (
 	"github.com/Craftserve/potoq"
 	"github.com/Craftserve/potoq/packets"
 	"github.com/Craftserve/potoq/utils"
-
-	l4g "github.com/alecthomas/log4go"
 )
 
 var Formatter func(recipient, sender *potoq.Handler, msg string) string = DefaultFormatter
 var MessageHook func(handler *potoq.Handler, kind, message string)
 var globalChatConfig *chatConfig
 var globalChatLock sync.Mutex
 var globalChatLimiter *rate.Limiter
-var log l4g.Logger
-var supervisor_log l4g.Logger
+var chat_log *logrus.Logger
+var supervisor_log *logrus.Logger
```

Comment:
Use camelCase instead of snake_case.

Reply:
No jak już to przerabiamy to można faktycznie